markernum: 1
# rotation and translate from frame to marker
#0.1848+0.11684+0.05974 0.67815; -0.072639; -0.16025
# measured_translation: [0.794, -0.043, 0.214]

measured_translation: [0.766, 0.328, 0.205]
measured_rot: [1, 0, 0, 0, 1, 0, 0, 0, 1]
frame: "/left_gripper_base"

#For display purposes only
squaredims: [0.07, 0.04, 0.02]

